app.js

```diff
@@ -77,10 +77,8 @@ if (app.get('env') == 'development') {
 }
 
 // Uncomment code below to create vulnerability - Hardcoded values
-/*
 var token = 'SECRET_TOKEN_f8ed84e8f41e4146403dd4a6bbcea5e418d23a9';
```

Comment:
### <sub><img src="https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-high.svg" width="20"/></sub>&nbsp; Hardcoded Secret

Avoid hardcoding values that are meant to be secret. Found a hardcoded string used in here.

<h6>
<a href="https://cwe.mitre.org/data/definitions/547.html">CWE-547</a> | <a href="https://docs.snyk.io/manage-risk/prioritize-issues-for-fixing/priority-score">Priority score 806</a> | Line 80</h6>

<details>

<summary><b>Data flow: 2 steps</b></summary>

`Step 1` https://github.com/trailblazing-dx/tiago_app.dev.snyk_github_oauth_nodejs-goof/blob/413bfc39ffab0fb15bbe1538685b6337a19a1563/app.js#L80

`Step 2` https://github.com/trailblazing-dx/tiago_app.dev.snyk_github_oauth_nodejs-goof/blob/413bfc39ffab0fb15bbe1538685b6337a19a1563/app.js#L80

</details>